classifiers/sentiment/vader_sentiment_classifier/code_snippet_refinery.md

```diff
@@ -2,7 +2,7 @@
 # expects labeling task to have labels ["positive" ,"neutral", "negative"]
 from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
 
-ATTRIBUTE: str = "text" # only text attributes
+ATTRIBUTE: str = "headline" # only text attributes
```

Comment:
why was this changed to headline? we usually use text everywhere else

Reply:
Yes this is correct. This shouldn't be here, anyway. Removed it!